verification/verify_html_to_rss.py

```
@@ -0,0 +1,85 @@
+        step3.locator("button:has-text('下一步')").click()
+
+        expect(recipe_id).to_have_value("test-feed-title")
+
+        step4.locator(".arco-input-append button").click()
+
+        expect(recipe_id).to_have_value("updated-title-123")
+
+        print("Taking success screenshot...")
+        page.screenshot(path="verification/verification.png")
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)